settings.py

```diff
@@ -752,7 +752,7 @@ def _color_scheme_completions(view):
                 "{}  \tPackage: {}".format(file_name, package),
                 scheme_path
             )
-            completions.add(item)
+            completions.append(item)
         return completions
```

Comment:
`sorted_completions` returns a list and should have been used here. Seems I just forgot to add it.